cmd/hope/node/init.go

```diff
@@ -50,15 +51,15 @@ var initCmd = &cobra.Command{
 			return err
 		}
 
+		var lbp *hope.Node = nil
+		var nnf *utils.NodeNotFoundError
 		loadBalancer, err := utils.GetLoadBalancer()
-		if err != nil && loadBalancer != (hope.Node{}) {
```

Comment:
Had to make this change because the string slice on `hope.Node` prevents these simple comparisons.